sparkifydb/domain/model.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
+"""This module contains object representing business logic.
+
+Created on: 29/6/22
+@authors:
+    - Heber Trujillo <heber.trj.urt@gmail.com>
+    - Montse Navarro <montserrat.nvro.lpz@gmail.com>
+Licence,
+"""
+from dataclasses import dataclass, field
+from typing import Optional
+
+@dataclass(order=True, frozen=True)
```

Comment:
Thanks for your work. I have two comments/doubts:

1  Why did you use the "order" argument? 
2  I recently encounter [an error](https://github.com/cosmicpython/code/issues/17) when using a frozen dataclass in as part of the SQLAlchemy's ORM; I'd suggest using `unsafe_hash=True` instead of `frozen=True`.

Reply:
Hi Heber, thanks for your review. I use the order argument because I thought it was good practice to be able to compare two records. 
Regarding the second point, I have been the changes.